backend/index.js

```diff
@@ -28,6 +28,7 @@ mongoose.connection.on('error', dbConnectionFailure);
  * Error handler
  */
 const errorHandler = (err, req, res, next) => {
+  console.log(err);
```

Comment:
Why do we console.log here and additionally down in line 49? Should this be condensed into one logging statement?